packages/gestalt/src/IconButtonFloating.js

```diff
@@ -0,0 +1,69 @@
+// @flow strict
+import { forwardRef, type Node, useImperativeHandle, useRef } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import icons from './icons/index.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
```

Comment:
We can't use this in any user-facing types as it is opaque to the user. The event type needs to be defined explicitly